openpype/modules/deadline/abstract_submit_deadline.py

```diff
@@ -627,7 +628,14 @@ def submit(self, payload):
             self.log.debug(payload)
             raise RuntimeError(response.text)
 
-        result = response.json()
+        try:
+            result = response.json()
+        except json.decoder.JSONDecodeError:
+            msg = "Broken response {}. ".format(response)
+            msg += "Try restarting DL webservice"
+            self.log.warning()
```

Comment:
This seems to be missing the message.
```suggestion
            self.log.warning(msg)
```

Reply:
I did small edit in the suggestion to log traceback too.